tox.ini

```diff
@@ -0,0 +1,67 @@
+[tox]
+envlist = flake8,pydocstyle,docs
+
+
+[testenv]
+basepython=python2.7
+# Installing with sdist makes tox/pip very unhappy.
+usedevelop=True
+# Bodhi requires koji, which isn't available on PyPI.
+sitepackages=True
+
+
+[testenv:docs]
+changedir = docs
+deps =
+    sphinx
+    cornice_sphinx
+whitelist_externals =
+    mkdir
+    rm
+    sphinx-build
+commands=
+    rm -rf _build images
+    mkdir -p _static images
+    ./generate_db_schema
+    sphinx-build -W -b html -d {envtmpdir}/doctrees .  _build/html
+    sphinx-build -W -b man -d {envtmpdir}/doctrees .  _build/man
```

Comment:
Very nice! You could add docs/_build/* as a Jenkies artifact too if you want. That would go here:

https://github.com/fedora-infra/bodhi/blob/93fad0c227539bfb6a759d7bcca8bb726619353a/devel/ci/githubprb-project.yml#L47-L48

If you do that, you'll also need to rsync them back after the tests run. You'd do that here:

https://github.com/fedora-infra/bodhi/blob/93fad0c227539bfb6a759d7bcca8bb726619353a/devel/ci/githubprb-project.yml#L88-L89

Completely optional for this pull request, but I'll probably do it if you don't because it seems like it'd be nice ☺